src/main/resources/db/changelog/0000300_tesler_core.xml

```diff
@@ -751,7 +751,7 @@
       <column name="DESCRIPTION" type="CLOB"/>
       <column name="SNIPPET" type="CLOB"/>
       <column name="PAGE_LIMIT" remarks="Лимит" type="NUMBER(10, 0)"/>
-      <column defaultValueNumeric="1" name="GRID_WIDTH" type="NUMBER(1, 0)">
+      <column defaultValueNumeric="1" name="GRID_WIDTH" type="NUMBER(2, 0)">
```

Comment:
@achabalov I'm not sure whether it legal. @IgorNB could you check it out please?

Reply:
It's ok